src/Mod/Spreadsheet/TestSpreadsheet.py

```diff
@@ -33,354 +33,561 @@
+
+        cls.sheet.set("G1", "=average(C13:C15)")
+        cls.sheet.set("G2", "=min(C13:C15)")
+        cls.sheet.set("G3", "=max(C13:C15)")
+        cls.sheet.set("G4", "=count(C13:C15)")
+        cls.sheet.set("G5", "=stddev(C13:C15)")
+        cls.sheet.set("G6", "=sum(C13:C15)")
+
+        cls.sheet.set("H1", "=average(C13:C16)")
+        cls.sheet.set("H2", "=min(C13:C16)")
+        cls.sheet.set("H3", "=max(C13:C16)")
+        cls.sheet.set("H4", "=count(C13:C16)")
+        cls.sheet.set("H5", "=stddev(C13:C16)")
+        cls.sheet.set("H6", "=sum(C13:C16)")
+
+        cls.doc.recompute()
+
+    @classmethod
+    def tearDownClass(cls):
+        FreeCAD.closeDocument(cls.doc.Name)
+
+    def test_values(self):
+        self.assertEqual(self.sheet.A1, 1)
+        self.assertEqual(self.sheet.A2, 3)
+        self.assertEqual(self.sheet.A3, 6)
+        self.assertEqual(self.sheet.A4, 10)
+        self.assertEqual(self.sheet.A5, 21)
+
+    def test_sum(self):
+        self.assertEqual(self.sheet.B1, 1)
+        self.assertEqual(self.sheet.B2, 1)
+        self.assertEqual(self.sheet.B3, 1)
+        self.assertEqual(self.sheet.B4, 1)
+        self.assertEqual(self.sheet.B5, 1)
+
+    def test_min(self):
+        self.assertEqual(self.sheet.C1, 1)
+        self.assertEqual(self.sheet.C2, 2)
+        self.assertEqual(self.sheet.C3, 3)
+        self.assertEqual(self.sheet.C4, 4)
+        self.assertEqual(self.sheet.C5, 6)
+
+    def test_max(self):
         self.assertTrue(
-            sheet.D1.startswith("ERR: Invalid number of entries: at least two required.")
+            self.sheet.D1.startswith("ERR: Invalid number of entries: at least two required.")
         )
-        self.assertEqual(sheet.D2, 0.7071067811865476)
-        self.assertEqual(sheet.D3, 1.0)
-        self.assertEqual(sheet.D4, 1.2909944487358056)
-        self.assertEqual(sheet.D5, 1.8708286933869707)
-
-        self.assertEqual(sheet.E1, 1)
-        self.assertEqual(sheet.E2, 2)
-        self.assertEqual(sheet.E3, 3)
-        self.assertEqual(sheet.E4, 4)
-        self.assertEqual(sheet.E5, 6)
-
-        self.assertEqual(sheet.F1, 1)
-        self.assertEqual(sheet.F2, (1.0 + 2.0) / 2.0)
-        self.assertEqual(sheet.F3, (1.0 + 2 + 3) / 3)
-        self.assertEqual(sheet.F4, (1.0 + 2 + 3 + 4) / 4)
-        self.assertEqual(sheet.F5, (1.0 + 2 + 3 + 4 + 5 + 6) / 6)
-
-        self.assertEqual(sheet.G1, Units.Quantity("5 mm"))
-        self.assertEqual(sheet.G2, Units.Quantity("4 mm"))
-        self.assertEqual(sheet.G3, Units.Quantity("6 mm"))
-        self.assertEqual(sheet.G4, 3)
-        self.assertEqual(sheet.G5, Units.Quantity("1 mm"))
-        self.assertEqual(sheet.G6, Units.Quantity("15 mm"))
 
+    def test_stddev(self):
+        self.assertEqual(self.sheet.D2, 0.7071067811865476)
+        self.assertEqual(self.sheet.D3, 1.0)
+        self.assertEqual(self.sheet.D4, 1.2909944487358056)
+        self.assertEqual(self.sheet.D5, 1.8708286933869707)
```

Comment:
the `D1` test is the first part of the `test_stddev` test. By merging the two you will be in sync again.
```py
    def test_stddev(self):
        self.assertTrue(
            self.sheet.D1.startswith("ERR: Invalid number of entries: at least two required.")
        )
        self.assertEqual(self.sheet.D2, 0.7071067811865476)
        self.assertEqual(self.sheet.D3, 1.0)
        self.assertEqual(self.sheet.D4, 1.2909944487358056)
        self.assertEqual(self.sheet.D5, 1.8708286933869707)
```

But as mentioned in first comment, this issue is due to the setup code for each test being placed outside the test unit.